package.json

```diff
@@ -48,5 +48,5 @@
       "fast-xml-parser@<4.4.1": ">=4.4.1"
     }
   },
-  "packageManager": "pnpm@8.15.7"
+  "packageManager": "pnpm@9.5.0"
```

Comment:
I think we can remove this and allow toolings to use their latest PNPM 9 minor/patch versions